packages/playwright-core/src/server/network.ts

```diff
@@ -265,9 +265,18 @@ export class Route extends SdkObject {
         isBase64 = false;
       }
     }
+    const headers = [...(overrides.headers || [])];
+    if (overrides.cors !== false) {
```

Comment:
I guess we never arrive here for preflights? If we do, we might need some special handling.

Reply:
Added a test, we don't currently intercept preflights in any browser.